Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianCKCAlgorithm.H

```diff
@@ -108,6 +108,14 @@ struct CartesianCKCAlgorithm {
         return delta_t;
     }
 
+    /**
+     * Returns maximum number of guard cells required by the field-solve
```

Comment:
Not 100% sure, but I think we could need `\brief` (here and below) in order for Doxygen to recognize this as brief description, because in our Doxyfile we have
https://github.com/ECP-WarpX/WarpX/blob/7a1a10f5c711d064ec8a5a1caa9fdecd36bad162/Docs/Doxyfile#L190
```suggestion
     * \brief Returns maximum number of guard cells required by the field-solve
```

Reply:
I did this to be consistent with what was there for the other functions in this file. But I can also change the other comments in the other functions :)